qualtran/bloqs/arithmetic/__init__.py

```diff
@@ -39,3 +39,5 @@
     SumOfSquares,
 )
 from qualtran.bloqs.arithmetic.sorting import BitonicSort, Comparator
+
+from ._shims import *
```

Comment:
import * is sort of discouraged right?

Reply:
made explicit. these shims will be removed in follow-on PRs (replaced with real implementations)